apps/etterlatte-behandling/src/main/kotlin/grunnlagsendring/doedshendelse/kontrollpunkt/DoedshendelseKontrollpunktService.kt

```diff
@@ -95,6 +112,18 @@ class DoedshendelseKontrollpunktService(
         )
     }
 
+    private fun kontrollerAvdoedHarYtelseIGjenny(hendelse: DoedshendelseInternal): DoedshendelseKontrollpunkt.AvdoedHarYtelse? {
+        val sakerForAvdoed = sakService.finnSaker(hendelse.avdoedFnr)
+        return if (sakerForAvdoed.isEmpty()) {
+            null
+        } else {
+            sakerForAvdoed.forEach {
+                kontrollerEksisterendeHendelser(hendelse, it)
```

Comment:
Vil den returnere ut av denne `forEach`-en kun dersom det finnes en duplikat (som jeg antar er ønsket resultat her)? Hva hvis den returnerer null? Ønsker man evt. både duplikat + avdoedharytelse kontrollpunktene med videre?

Burde hatt en test på den logikken.

Reply:
Altså ja den går til siste i scopet for elsen.

Reply:
Er vel fint å få duplikatsjekken også? spesielt for de andre scenarioene

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/4053/commits/edfde15c5c1597dacc896aa76fc713c5b4fb05ca